### ISSUE TITLE
Question about test set

### ISSUE THREAD

**Issue by AyahSoufan** [zdmc23/sentiment-analysis-arabic#1]
Hello,

I have a question, from where you get the test set?
The data set that you mentioned you devided it by training and validation only, so from where you got the testing set?

Thanks
Ayah

**Comment by zdmc23** [comment]
Hello Ayah, 

I have a private test set that was actually Facebook post content.  The Twitter training/validation set was public, and I made the assumption that social media content would generalize well enough (there is a risk that Twitter content is significantly different than Facebook due to character restraints, but these particular FB posts were rarely more than 3 sentences because they were comments).

Regards,
Zach
